translations/translations.json

```diff
@@ -2338,6 +2338,7 @@
       "notSupportedByDevice": "Diese Funktion wird auf diesem Gerät nicht unterstützt.",
       "languageSwitchFailedTitle": "Leider ist der Sprachwechsel fehlgeschlagen.",
       "languageSwitchFailedMessage": "Die ausgewählte Sprache ist nicht offline verfügbar. Eine aktive Internetverbinding ist notwendig.",
+      "noPushNotificatidbonPermission": "Berechtigung erforderlich",
```

Comment:
```suggestion
      "pushNotificationPermissionRequired": "Berechtigung erforderlich",
```

Reply:
I would prefer to rename the key, if you disagree at least the typo needs to be fixed.